src/sentry/api/serializers/models/plugin.py

```diff
@@ -5,13 +5,27 @@
 from sentry.api.serializers import Serializer
 from sentry.utils.assets import get_asset_url
 from sentry.utils.http import absolute_uri
+from sentry.models import ProjectOption
 
 
 class PluginSerializer(Serializer):
     def __init__(self, project=None):
         self.project = project
 
     def serialize(self, obj, attrs, user):
+        from sentry.api.endpoints.project_releases_token import _get_webhook_url
+        doc = ''
+
+        release_token = ProjectOption.objects.get_value(self.project, 'sentry:release-token')
```

Comment:
so would this only be `None` if they'd never even visited the release tracking page?